uaclient/cli.py

```diff
@@ -211,7 +211,7 @@ def action_enable(args, cfg):
 
     @return: 0 on success, 1 otherwise
     """
-    logging.debug(ua_status.MESSAGE_REFRESH_ENABLE)
+    logging.info(ua_status.MESSAGE_REFRESH_ENABLE)
```

Comment:
We should be using `print` for messages we explicitly want users to see, so that we can eventually disable logging output to the console entirely.

Reply:
done! thanks